Symfony/CS/ConfigInterface.php

```diff
@@ -128,4 +137,13 @@ public function getCacheFile();
      * @return string|null
      */
     public function getPhpExecutable();
+
+    /**
+     * Set the fixers to be used.
+     *
+     * @param FixerInterface[] $fixers
```

Comment:
The user must set names, and just our code that run must have instances. So maybe we should keep that instances somewhere else